packages/headless/src/controllers/product-recommendations/headless-frequently-viewed-together.test.ts

```diff
@@ -28,4 +28,14 @@ describe('headless frequently-viewed-together', () => {
     engine.state.productRecommendations.isLoading = true;
     expect(frequentlyViewedTogether.state.isLoading).toBe(true);
   });
+  it('object shape matches original', () => {
+    expect(frequentlyViewedTogether.refresh).toBeTruthy();
+    expect(frequentlyViewedTogether.setSkus).toBeTruthy();
+    expect(frequentlyViewedTogether.subscribe).toBeTruthy();
+    expect(frequentlyViewedTogether.state.error).toBeFalsy();
+    expect(frequentlyViewedTogether.state.isLoading).toBeFalsy();
+    expect(frequentlyViewedTogether.state.maxNumberOfRecommendations).toBe(5);
+    expect(frequentlyViewedTogether.state.recommendations).toBeTruthy();
+    expect(frequentlyViewedTogether.state.skus).toBeTruthy();
+  });
```

Comment:
LGTM, tho in the future @coveo/search, maybe we should think about using Zod or Bueno to better transcribe the obj structure?